archiving-of-web-and-social-media.py

```diff
@@ -95,6 +95,19 @@ def prepare_and_clean_columns_and_index(data):
     return data
 
 
+def get_timeout(platform):
+    default = 30
+    timeouts = {
+        "gislaved.se": 10,
+        "insidan.gislaved.se": 10,
+        "facebook": 20,
+        "instagram": 30,
+        "linkedin": 20,
+        "general":20

```

Comment:
Vad menas med den som heter 'general'? Känns som att det blir en dubbel defaultvärde?

Såg att du använder den längre ner i funktionen som inkluderar flera olika hemsidor. Tänker att du istället för att ha en general bara anropar på funktionen utan något värde och får default tillbaks. Om du tror att 20 är tillräckligt så kan du ändra till det. Skrev längre ner hur jag gjorde valet av tider. Men om det aldrig tagit ens nästan så lång tid när du har kört programmet så är lite kortare tid nog okej.

Reply:
Jag är väldigt osäker på hur man ska göra i denna situation för det har funkat förut med ingen wait och jag vet inte om det behövs på alla och varför jag har haft det på vissa och inte på andra. Gissar att det är en trail and error från min sida i programmering tills det har funkat. Det som du säger med Instagram 20  t ex är lite begränsande för det kan ju vara så att man vill finjustera det så att element har 5 och den andra 20 för att trimma och få en effektiv web extrahering. Det blir ju ganska trögt att köra  t ex 100 url:er om man måste vänta 20 plus 20 plus 20 för varje url.  Sedan är det ju också så att instagram 20 kanske funkar i inloggningen men inte när man kör klicka på cookie banner i Instgram get_screenshot. Jag gjorde general för att jag inte vet hur man ska göra för att få till detta så att det både blir programmerinsmässigt snyggt och även praktiskt användbart när man ska köra programmet så att det inte blir segt.